File: CS/ShowFilterPopupListBox example/Form1.cs
using System;
using System.Windows.Forms;
using DevExpress.XtraTreeList;

namespace ShowFilterPopupListBox_example
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void treeList1_ShowFilterPopupListBox(object sender, FilterPopupListBoxEventArgs e)
        {
            if (e.Column.FieldName == "Name")
            {
                //Removes "(Blanks)"
                e.ComboBox.Items.RemoveAt(0);
                //Removes "(Non blanks)"
                e.ComboBox.Items.RemoveAt(0);
            }
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            this.table1TableAdapter.Fill(this.treelistdbDataSet.Table1);

        }
    }
}

File: VB/ShowFilterPopupListBox example/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Windows.Forms
Imports DevExpress.XtraTreeList

Namespace ShowFilterPopupListBox_example
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub treeList1_ShowFilterPopupListBox(ByVal sender As Object, ByVal e As FilterPopupListBoxEventArgs) Handles treeList1.ShowFilterPopupListBox
			If e.Column.FieldName = "Name" Then
				'Removes "(Blanks)"
				e.ComboBox.Items.RemoveAt(0)
				'Removes "(Non blanks)"
				e.ComboBox.Items.RemoveAt(0)
			End If
		End Sub

		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			Me.table1TableAdapter.Fill(Me.treelistdbDataSet.Table1)

		End Sub
	End Class
End Namespace
